scheduler/test/cook/test/scheduler/scheduler.clj

```diff
@@ -129,17 +176,34 @@
                       :framework-id framework-id
                       :slave-id (mtypes/map->SlaveID {:value slave-id})
                       :hostname hostname
-                      :resources (make-offer-resources cpus mem disk ports gpus)
-                      :attributes (make-offer-attributes (merge attrs {"HOSTNAME" hostname}))
+                      :resources (make-mesos-offer-resources cpus mem disk ports gpus)
+                      :attributes (make-mesos-offer-attributes (merge attrs {"HOSTNAME" hostname}))
                       :executor-ids []}))
 
-(defn make-vm-offer
+(defn make-mesos-vm-offer
   [framework-id host offer-id & {:keys [attrs cpus mem disk] :or {attrs {} cpus 100.0 mem 100000.0 disk 100000.0}}]
   (sched/->VirtualMachineLeaseAdapter
     (make-mesos-offer offer-id framework-id "test-slave" host
                       :cpus cpus :mem mem :disk disk :attrs attrs) 0))
 
-;(.getOffer (make-vm-offer (make-uuid) "lol" (make-uuid)))
+(defn make-k8s-offer
+  [id framework-id slave-id hostname & {:keys [cpus mem disk ports gpus attrs]
+                                        :or   {cpus 40.0 mem 5000.0 disk 6000.0 ports {:begin 31000 :end 32000} gpus {} attrs {}}}]
+  {:id {:value id}
+   :framework-id framework-id
+   :slave-id {:value slave-id}
+   :hostname hostname
+   :resources (make-k8s-offer-resources cpus mem disk ports gpus)
+   :attributes (make-k8s-offer-attributes (merge attrs {"HOSTNAME" hostname}))
+   :executor-ids []})
+
+(defn make-k8s-vm-offer
+  [framework-id host offer-id & {:keys [attrs cpus mem gpus disk] :or {attrs {} cpus 100.0 mem 100000.0 gpus {} disk 100000.0}}]
+  (sched/->VirtualMachineLeaseAdapter
+    (make-k8s-offer offer-id framework-id "test-slave" host
+                    :cpus cpus :mem mem :gpus gpus :disk disk :attrs attrs) 0))
+
+;(.getOffer (make-mesos-vm-offer (make-uuid) "lol" (make-uuid)))
```

Comment:
Delete?

Reply:
I didn't add this so I was just going to leave it there. Should I still delete?

Reply:
I think so